runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -3251,7 +3251,7 @@ class INTERPRETER_CLASS
 	VMINLINE VM_BytecodeAction
 	inlReferenceGetImpl(REGISTER_ARGS_LIST)
 	{
-		/* Only called from java for metronome GC policy */
+		/* Only called from java for SATB Barrier */
```

Comment:
we call this always now, right? then we need to remove this comment

Reply:
Correct! will remove comment

Reply:
Comment removed from here and `Fast_java_lang_ref_Reference_getImpl`